yarn.lock

```diff
@@ -4977,22 +4963,22 @@ prosemirror-history@~1.3.0:
     prosemirror-transform "^1.0.0"
     rope-sequence "^1.3.0"
 
-prosemirror-keymap@~1.2.0:
+prosemirror-keymap@^1.2.0:
   version "1.2.0"
   resolved "https://registry.yarnpkg.com/prosemirror-keymap/-/prosemirror-keymap-1.2.0.tgz#d5cc9da9b712020690a994b50b92a0e448a60bf5"
   integrity sha512-TdSfu+YyLDd54ufN/ZeD1VtBRYpgZnTPnnbY+4R08DDgs84KrIPEPbJL8t1Lm2dkljFx6xeBE26YWH3aIzkPKg==
   dependencies:
     prosemirror-state "^1.0.0"
     w3c-keyname "^2.2.0"
 
-prosemirror-model@^1.0.0, prosemirror-model@^1.16.0, prosemirror-model@~1.18.3:
+prosemirror-model@^1.0.0, prosemirror-model@^1.16.0, prosemirror-model@^1.18.3:
```

Comment:
What dependency brings in the prosemirror-model@^1.0.0? This seems to be entirely incorrect. Can it be fixed as a part of this update?

Reply:
All versions other than "^1.18.3" are coming from other prosemirror dependencies. e.g. "^1.0.0" is coming from prosemirror-commands. Since everything is specified as peer dependency it should be fine.